contrib/isso-dev.cfg

```diff
@@ -37,8 +37,9 @@ reply-to-self = true
 [markup]
 options = autolink, fenced-code, no-intra-emphasis, strikethrough, superscript
 flags =
-allowed-elements =
-allowed-attributes =
+allowed-elements = a, p, hr, br, ol, ul, li, pre, code, blockquote, del, ins,
```

Comment:
Please add `tr` to the allowed elements in a separate commit, it's not reasonable that we allow all other table elements but this one